### DISCUSSION TITLE
SX127x Specified device

### DISCUSSION THREAD

**Discussion by klaids** [jgromes/RadioLib#366] [General]
Hi, 
I have a Lora Module with a SX1272 chip. I want to realize an application where two device communicate together. I don't understand if/how is possibile, with the RadioLib library set an address to each node and transmit a message to a specific node. 
Best Regards

**Comment by jgromes** [comment]
LoRa by itself has no concept of "node address", if you need that sort of thing then you have to implement it yourself, for example have the first byte of the message determine address and only accept messages that match the expected node address.

If instead of LoRa you choose to use FSK, then that can do this automatically.
